src/grt/src/fastroute/src/maze3D.cpp

```diff
@@ -142,12 +142,14 @@ void FastRouteCore::setupHeap3D(int netID,
   dest_heap_3D.clear();
 
   if (num_terminals == 2) {  // 2-pin net
-    d1_3D[0][y1][x1] = 0;
-    directions_3D[0][y1][x1] = Direction::Origin;
-    src_heap_3D.push_back(&d1_3D[0][y1][x1]);
-    d2_3D[0][y2][x2] = 0;
-    directions_3D[0][y2][x2] = Direction::Origin;
-    dest_heap_3D.push_back(&d2_3D[0][y2][x2]);
+    const int nt1 = treenodes[n1].stackAlias;
```

Comment:
Could you comment on why the stackAlias is used here? Also, a more meaningful name than `nt1` would make it clearer.

Reply:
The stackAlias is used the get the correct layer in case there is more than one node in the same GCell, so with this it is to get the correct layer and ensure that both pins will be covered. As for the name I just followed the pattern used in maze3D but I can change it.

Reply:
The names in maze3D could be better. I think you can change these var names here, and at some point, we should review the remaining of this file to improve its readability.